samples/simpleasset/simpleasset.go

```diff
@@ -0,0 +1,238 @@
+package simpleasset
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+	"os"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+// SmartContract provides functions for managing an Asset
+type SmartContract struct {
+	contractapi.Contract
+}
+
+type Asset struct {
```

Comment:
It looks like this is loosely meant to represent a bond? Why not call it a `BondAsset` then?